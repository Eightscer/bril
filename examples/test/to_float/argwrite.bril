@main(a: int) {
  cond: bool = const true;
  br cond .here .there;
.here:
  print a;
.there:
}
