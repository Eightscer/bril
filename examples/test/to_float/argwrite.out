@main(a: float) {
  cond: bool = const true;
  br cond .here .there;
.here:
  print a;
.there:
}
